packages/utils/src/crypto/ec-utils.ts

```diff
@@ -4,7 +4,7 @@ import { ethers } from 'ethers';
 /**
  * Function to manage Elliptic-curve cryptography
  */
-export default {
+export const EcUtils = {
```

Comment:
Can you please remove the named const here and export these functions individually like in the other utils modules? And then, can you please rename `recover` to `recoverSigner()`? The other functions don't need to be renamed.